zerver/lib/export.py

```diff
@@ -1266,11 +1266,11 @@ def convert_to_id_fields(data: TableData, table: TableName, field_name: Field) -
         item[field_name + "_id"] = item[field_name]
         del item[field_name]
 
-def re_map_foreign_keys(data: TableData,
-                        table: TableName,
+def re_map_foreign_keys(data_table: List[Record],
```

Comment:
We do this because we have to pass [records of avatars and attachments](https://github.com/zulip/zulip/pull/8683/commits/4f35f31b52d6d06bc0c42c0d2f678b5c4d9742d4) through the same function.

Reply:
I think to solve the issue this, we can make a low-level function `re_map_foreign_keys_internal` that takes `(data_table, table_name, ...)` and then in most cases just call a wrapper that does `re_map_foreign_keys_internal(data[table_name], table_name, ...)` but use the internal version from the avatar code path.

But a reasonable alternative might be to just update the print statement to not include the table name.